src/linear_algebra/general_linear_group.lean

```diff
@@ -164,4 +180,21 @@ lemma to_GL_pos_injective :
  from subtype.coe_injective).of_comp
 
 end special_linear_group
+
+section examples
+
+/-- The matrix [a, b; -b, a] (inspired by multiplication by a complex number); it is an element of
+`GL_2` if `a ^ 2 + b ^ 2` is nonzero. -/
```

Comment:
I don't think `GL_2` should be marked as code here, since we don't have anything called that. Maybe use latex instead via `$GL_2$` or `$\mathrm{GL}_2$` or however you would actually write in in LaTeX (I have no idea).